scrapers/koji.py

```diff
@@ -21,6 +21,7 @@ def run(self, since=None):
             since = utils.timestamp_to_datetime(since)
 
         self.builds = self.get_koji_builds(since)
```

Comment:
Can you also make this not be a class variable?

Reply:
Sure, Done!